Allow xset dpms settings to override .xscreensaver.

Upstream rejected:

You can't have *two* places that specify the same preference.
Therefore, xscreensaver should be the one and only place.


---
 driver/timers.c       |    3 +++
 driver/xscreensaver.c |    4 ++++
 2 files changed, 7 insertions(+), 0 deletions(-)

diff --git a/driver/timers.c b/driver/timers.c
--- a/driver/timers.c
+++ b/driver/timers.c
@@ -1549,6 +1549,8 @@ watchdog_timer (XtPointer closure, XtIntervalId *id)
 
   /* If the DPMS settings on the server have changed, change them back to
      what ~/.xscreensaver says they should be. */
+  /* mali- No!! if someone uses xset then let the changes be picked by
+   * xscreensaver...do not have to regulate everything from .xscreensaver.
   sync_server_dpms_settings (si->dpy,
                              (p->dpms_enabled_p  &&
                               p->mode != DONT_BLANK),
@@ -1556,6 +1558,7 @@ watchdog_timer (XtPointer closure, XtIntervalId *id)
                              p->dpms_suspend / 1000,
                              p->dpms_off / 1000,
                              False);
+  **/
 
   if (si->screen_blanked_p)
     {
diff --git a/driver/xscreensaver.c b/driver/xscreensaver.c
--- a/driver/xscreensaver.c
+++ b/driver/xscreensaver.c
@@ -1587,6 +1587,10 @@ main (int argc, char **argv)
   init_sigchld ();
 
   disable_builtin_screensaver (si, True);
+/*****
+  mali- this may need to be taken out in future if it hinders user
+  setting dpms values through xset.
+ *****/
   sync_server_dpms_settings (si->dpy,
                              (p->dpms_enabled_p  &&
                               p->mode != DONT_BLANK),

